### PR TITLE
[ADMIN] Use an AdminExtension to synchronise AMP content

### PR DESCRIPTION
**Pull request by Remg**

### CODE REVIEW

File: src/Form/EventListener/AmpSynchronisedListener.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    {
+        return [
+            FormEvents::SUBMIT => 'setAmpContent',
+        ];
+    }
+
+    public function setAmpContent(FormEvent $event): void
+    {
+        /** @var EntityContentInterface $entityContent */
+        if (!$entityContent = $event->getData()) {
```
Comment (ottaviano) [inline src/Form/EventListener/AmpSynchronisedListener.php]: `if (!$entityContent instanceof EntityContentInterface) { return; }`

File: app/config/services/admin.xml [hunk 2]
```diff
@@ -577,6 +561,14 @@
 
         <service autowire="true" public="false" id="AppBundle\Form\DataTransformer\CommitteeTransformer"/>
 
+        <service id="app.admin.amp_synchronised_extension" class="AppBundle\Admin\Extension\AmpSynchronisedAdminExtension">
+            <argument type="service" id="AppBundle\Form\EventListener\AmpSynchronisedListener" />
+
+            <tag name="sonata.admin.extension" target="app.admin.article" />
```
Comment (B-Galati) [inline app/config/services/admin.xml]: Clarfication admin is missing no ?

Comment (Remg) [reply app/config/services/admin.xml]: this admin is not declared at all. i guess we have to remove it completely :)

**Comment by thomasbisignani** [comment]
:ok_hand:

File: app/config/services/admin.xml (hunk 2 above)
Comment (Remg) [reply app/config/services/admin.xml]: @guillaumedenhez confirmed: we should remove the `ClarificationAdmin` code for now as it is useless.

**Pull request merged**